manage-participants/main.yml

```diff
@@ -3,6 +3,14 @@
 - hosts: identity-hosts
   name: Verify Identity Provider
   tasks:
+    - name: "Fail if ocp_subdomain or subdomain_base_suffix not provided"
+      fail:
+        msg: "ocp_subdomain and subdomain_base_suffix vars needs to be provided in order to continue withe the playbook"
+      when:
+        - ocp_subdomain is undefined or subdomain_base_suffix is undefined
```

Comment:
let's also check for "empty" variables here - i.e.: `ocp_subdomain|trim == ""` 

Also would be good to include the output of the variables in the message to know which one is empty - i.e.: something like the following:
```
msg: "Variable foo ({{ foo|default('')}}) or bar ({{ bar|default('')}}) is not defined or empty"
```

Reply:
Fixed in the 458b368afac80eb7e2806fd17cbbe2ecf34eb0f8 commit.